packages/components/src/components/popup/popup.tsx

```diff
@@ -76,7 +75,7 @@ const Popup = ({
                     </MobileFullPageModal>
                 </div>
             </PopupContext.Provider>,
-            document.getElementById('popup_root')
+            document.getElementById('popup_root')!
```

Comment:
as far as i know `non-null assertion operator` throws error in deriv-app ts configuration
please make sure that it is not throwing any warning/error

Reply:
It works properly